source/plugin/text/representations/configurate.rst

```diff
@@ -0,0 +1,47 @@
+====================
+Configuration Format
+====================
+
+SpongeAPI offers support to serialize text directly to a Configurate configuration file through the use of TypeTokens.
+Text are saved using the same node structure as the Text's JSON representation.
+
+.. tip::
+
+    For information on how to use Configurate to create configuration files for your plugin, please refer to
+    :doc:`/plugin/configuration/index`.
```

Comment:
This doesn't link correctly in the preview.

P.S: If you want to preview your changes before commiting them, follow the instructions on the [readme](https://github.com/SpongePowered/SpongeDocs/blob/master/README.md), if you haven't already.